Srigowri_Deogirikar/html4.html

```diff
@@ -16,6 +16,6 @@ <h4>Notepad or TextEdit </h4>
              <br>
              However, for learning HTML we recommend a simple text editor like Notepad(PC)or TextEdit(Mac)
             </p2>
```

Comment:
what is p2 tag

Reply:
Just for avoiding confusion with paragraph , i named it like that sir